### ISSUE TITLE
Pattern not initialized when adding a new row

### ISSUE THREAD

**Issue by frapell** [collective/collective.z3cform.datagridfield#198]
The condition used with `.find()` in https://github.com/collective/collective.z3cform.datagridfield/blob/dfdf2371af1315e504a6c411be80fa3793f7f51f/resources/js/datagridfield.pattern.js#L341-L345 require for the `pat-*` to be the very first class element, meaning that an element such as

```
<div class="custom-field pat-mypattern"></div>
```
When a new row is added will become

```
<div class="custom-field dgw-disabled-pat-mypattern"></div>
```

and doesn't get replaced, and so its pattern never initialized.
It will work fine, if instead the original widget was like

```
<div class="pat-mypattern custom-field "></div>
```

**Comment by frapell** [comment]
@petschki I want to submit a PR with this quick fix, should I submit it including the compiled resources? or is that done before making a new release?

**Comment by petschki** [comment]
With compiled resources please. The best is when the compiled resources are a single commit so it's easier to read the changes.

**Comment by petschki** [comment]
released `3.0.3`
